src/librustc_error_codes/error_codes/E0708.md

```diff
@@ -0,0 +1,26 @@
+async` non-`move` closures with parameters are not currently supported
+
+Erroneous code example:
+
+```compile_fail,E0710
+#![feature(async_closure)]
+
+fn main() {
+    let add_one = async |num: u8| {
+        num + 1
+    };
+}
+```
+
+`async` with non-move currently not supported with the current
+version, you can use successfully by using move:
+
+```
+#![feature(async_closure)]
+
+fn main() {
+    let add_one = async move |num: u8| {
```

Comment:
And just like you have to add `// error!`, you need to add a comment like `// ok!` on the fixed lines.